.github/workflows/ci.yml

```diff
@@ -35,6 +33,25 @@ jobs:
 
       - name: Lint using ESLint
         run: yarn lint .
+  test:
+    runs-on: ubuntu-18.04
```

Comment:
Does it want to be testing on the latest ubuntu version too?

Reply:
Yes thanks good spot.